sdcm/read_operation_thread.py

```diff
@@ -50,21 +51,24 @@ def randomly_add_timeout(cmd) -> str:
             cmd += cql_timeout_param
         return cmd
 
-    def randomly_form_cql_statement(self, ks_cf: str) -> str:
-        cmd = random.choice(self.query_options).format(ks_cf)
-        return self.randomly_add_timeout(cmd)
+    @abstractmethod
+    def randomly_form_cql_statement(self) -> Optional[str]:
+        ...
 
     def create_session(self, db_node: BaseNode):
         credentials = self.db_cluster.get_db_auth()
         username, password = credentials if credentials else (None, None)
         return self.db_cluster.cql_connection_patient(db_node, user=username, password=password)
 
-    def run_fullscan(self, db_node: BaseNode):  # pylint: disable=too-many-locals
-        ks_cf = self.get_ks_cs(db_node)
+    def run_read_operation(self, read_operation_event):  # pylint: disable=too-many-locals
```

Comment:
The name "read_operation" that you changed everywhere, isn't clear.
Change it back to fullscan.
The reversed query is a partition full scan, so you can treat it as such.
We should have "full_partition_scan" and "full_partition_scan_reveresed_order" (aka revresed_query).

Reply:
@roydahan , how about renaming to just "scan_operation"? it may be less confusing

Reply:
ok.

Reply:
fixed